plugins/auth-backend/api-report.md

```diff
@@ -718,4 +740,9 @@ export type WebMessageResponse =
       type: 'authorization_response';
       error: Error;
     };
+
+// Warnings were encountered during analysis:
+//
+// src/providers/cloudflare-access/provider.d.ts:77:5 - (ae-forgotten-export) The symbol "CloudflareAccessClaims" needs to be exported by the entry point index.d.ts
+// src/providers/cloudflare-access/provider.d.ts:78:5 - (ae-forgotten-export) The symbol "CloudflareAccessIdentityProfile" needs to be exported by the entry point index.d.ts
```

Comment:
Please avoid adding new warnings to the API report, especially these ones mean we're not tracking the public API of these types. In this case you need to make sure these types are exported